src/charm.py

```diff
@@ -635,13 +610,19 @@ def _wp_generate_password(length=64):
             characters = string.ascii_letters + "!@#$%^&*()" + "-_ []{}<>~`+=,.;:/?|"
             return "".join(secrets.choice(characters) for _ in range(length))
 
-        return {
+        wp_secrets = {
             field: _wp_generate_password()
             for field in self._wordpress_secret_key_fields()
         }
+        wp_secrets["default_admin_password"] = secrets.token_urlsafe(32)
+        return wp_secrets
 
     def _replica_relation_data(self):
```

Comment:
Do you think a docstring would be helpful here?

Reply:
Added, thanks!

Reply:
I'm guessing the docstring is on its way, otherwise I'm good with this